stock/templates/public/characternotexisting.html

```diff
@@ -0,0 +1,90 @@
+{% load staticfiles %}
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <meta name="author" content="">
+
+    <title>Fleet participation</title>
+
+    <!-- Bootstrap Core CSS -->
+    <link href="{% static 'css/bootstrap.min.css' %}" rel="stylesheet">
+    <!-- Custom Fonts -->
+    <link href="{% static 'css/font-awesome.min.css' %}" rel="stylesheet" type="text/css">
+    <link href="{% static 'css/sb-admin-2.css' %}" rel="stylesheet">
+    {% block extra_css %}{% endblock extra_css %}
+    <style>
+        .grayiconecolor {
+            color: #505050;
+        }
+    </style>
+</head>
+
+<body onload="CCPEVE.requestTrust('http://www.mywebsite.com')">
```

Comment:
Ahhhhhhh................. I see why there where problems requesting trust now................................ This is just plain stupidity from my side.